udata/commands/fixtures.py

```diff
@@ -199,6 +198,9 @@ def import_fixtures(source):
                         MessageDiscussionFactory(**message, posted_by=user) for message in messages
                     ],
                 )
+            if "dataservices" not in fixture:
+                # For backwards compatibility.
+                continue
             for dataservice in fixture["dataservices"]:
```

Comment:
Maybe use dict.get ?
```suggestion
            # We may be missing dataservices in older fixtures
            for dataservice in fixture.get("dataservices", []):
```

Reply:
yep, even better, thanks 🙏